experiments/Cargo.toml

```diff
@@ -29,7 +29,12 @@ jexl-eval = "0.1.6"
 uuid = { version = "0.8", features = ["serde", "v4"]}
 sha2 = "0.9"
 hex = "0.4"
+# We use a fixed version of ffi-support to resolve the conflict between viaduct and uniffi in the versions of ffi_support they use
+ffi-support = "0.4.2"
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", branch = "main" }
```

Comment:
a bit dangerous to lock to a `main` branch, I am not sure if we should lock it to a GIT SHA at this point or not.

Reply:
opened https://github.com/mozilla/uniffi-rs/issues/253 to publish the uniffi crates to crates.io, for the time being, I can lock it to the latest commit